client/profile-wizard/steps/business-details/flows/selective-bundle/index.js

```diff
@@ -237,6 +237,21 @@ class BusinessDetails extends Component {
 			);
 		}
 
+		if (
+			! values.number_employees.length &&
+			[
+				'other',
+				'brick-mortar',
+				'brick-mortar-other',
+				'other-woocommerce',
+			].includes( values.selling_venues )
+		) {
```

Comment:
We should at some point create a config for all these fields, with it's dependencies, remove a bunch of this individual logic for each field. But that can be done at a later time.